chpl/chpl-service/src/main/java/gov/healthit/chpl/FeatureList.java

```diff
@@ -1,15 +1,23 @@
 package gov.healthit.chpl;
 
 /**
- * All FF4J feature flags.
+ * All relevant FF4J feature flags.
  * @author alarned
  *
  */
-public class FeatureList {
+public final class FeatureList {
+    private FeatureList() {}
+
+    /** Feature related to submitted Complaints. */
+    public static final String COMPLAINTS = "complaints";
```

Comment:
I think the complaints flag has been deployed to STG, so shouldn't 'be here

Reply:
removed